### PR TITLE
Feature/vk dz1 2

### PR DESCRIPTION
**Pull request by SamsonIOS**
https://user-images.githubusercontent.com/112130762/203489631-5fea2a54-669d-40de-9286-02f700ebd1c9.mp4

### CODE REVIEW

File: VK_App/NetworkManager/NetworkManager.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        static let version = "&v=5.81"
+    }
+
+    // MARK: - Public Methods
+
+    func getFriends() {
+        let path = "\(Constants.getFriends)\(Constants.acessToken)\(Constants.friendFields)\(Constants.version)"
+        let url = "\(Constants.baseURL)\(path)"
+        AF.request(url).responseJSON { response
+            in
```
Comment (Renbern) [inline VK_App/NetworkManager/NetworkManager.swift:29]: перенеси на 28

File: VK_App/NetworkManager/NetworkManager.swift [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        let path =
+            "\(Constants.getGroups)\(Constants.acessToken)\(Constants.friendFields)\(Constants.version)"
+        let url = "\(Constants.baseURL)\(path)"
+        AF.request(url).responseJSON { response in
+            guard let value = response.value else { return }
+            print(value)
+        }
+    }
+
+    func getGroups(group: String) {
```
Comment (Renbern) [inline VK_App/NetworkManager/NetworkManager.swift:55]:
```suggestion
    func getSearchedGroup(group: String) {
```

File: VK_App/VK_App/ViewControllers/FirstBarVC/FriendsTableViewController.swift [hunk 3]
```diff
@@ -72,16 +73,26 @@ final class FriendsTableViewController: UITableViewController {
...
+        infoAboutUser()
     }
 
     // MARK: Private Methods
 
+    private func infoAboutUser() {
+        networkManager.getFriends()
+        networkManager.getUserPhotos()
+        networkManager.getGroups()
+        networkManager.getGroups(group: Constants.searchGroupName)
```
Comment (Renbern) [inline VK_App/VK_App/ViewControllers/FirstBarVC/FriendsTableViewController.swift:93]:
```suggestion
        networkManager.getSearchedGroup(group: Constants.searchGroupName)
```

File: VK_App/VK_App/ViewControllers/LoginViewController.swift [hunk 4]
```diff
@@ -17,12 +47,15 @@ final class LoginViewController: UIViewController {
     private var secondDoteView = UIView()
     private var thirdDoteView = UIView()
 
+    private var networkManager = NetworkManager()
```
Comment (death654321) [inline VK_App/VK_App/ViewControllers/LoginViewController.swift:50]:
```suggestion
    private var networkAPIService = NetworkManager()
```

File: VK_App/VK_App/ViewControllers/LoginViewController.swift [hunk 5]
```diff
@@ -78,6 +111,28 @@ final class LoginViewController: UIViewController {
         scrollView.scrollIndicatorInsets = UIEdgeInsets.zero
     }
 
+    private func setUrlVK() {
```
Comment (death654321) [inline VK_App/VK_App/ViewControllers/LoginViewController.swift:114]:
```suggestion
    private func showAuthorizationWebView() {
```

**Review by death654321 — APPROVED** [review]

**Pull request merged**